hack/perf/helpers.bash

```diff
@@ -0,0 +1,36 @@
+ENGINE_A=${ENGINE_A:-/usr/bin/podman}
+ENGINE_B=${ENGINE_B:-/usr/bin/docker}
```

Comment:
Why not just keep set `podman`/`docker` as default instaed of the full path?. In this case it will use the one from $PATH, no?

Reply:
Nice idea!